integration_test/third_party_apps_data/test_config.yaml

```diff
@@ -1,5 +1,13 @@
 per_application_overrides:
   apache:
+    # TODO: reenable distros incrementally.
+    platforms_to_skip:
+      - centos-7
+      - centos-8
+      - rocky-linux-8
+      - sles-12
+      - sles-15
+  cassandra:
     # Skip all platforms listed above except debian-10 for now.
```

Comment:
Does it make sense to add some variant of this comment before line 3?

Reply:
moved comment